dev/src/validate.ts

```diff
@@ -151,6 +152,40 @@ export function validateString(
   }
 }
 
+/**
+ * Validates that 'value' is a host.
+ *
+ * @private
+ * @param arg The argument name or argument index (for varargs methods).
+ * @param value The input to validate.
+ * @param options Options that specify whether the host can be omitted.
+ * whether it can contain a port.
```

Comment:
sentence fragment: is "whether it can contain a port" supposed to be with options?

Reply:
Done